app/components/base/FundsFlowSelector.tsx

```diff
@@ -108,32 +108,6 @@ const customStyles = {
   }),
 };
 
-const senderTypeOptions = [
-  { value: "account", label: "Account" },
-  { value: "vcr", label: "Verified Customer" },
-  { value: "cr", label: "Unverified Customer" },
-];
-
-const senderSourceOptions = [
-  { value: "balance", label: "Balance" },
-  { value: "bank", label: "Bank" },
-  { value: "r01", label: "R01-Bank" },
-];
-
-const receiverTypeOptions = [
-  { value: "account", label: "Account" },
-  { value: "vcr", label: "Verified Customer" },
-  { value: "cr", label: "Unverified Customer" },
-  { value: "ro", label: "Receive-Only User" },
-];
-
-const receiverDestinationOptions = [
-  { value: "balance", label: "Balance" },
-  { value: "bank", label: "Bank" },
-  { value: "r03", label: "R03-Bank" },
-  { value: "card", label: "Debit Card" },
-];
-
```

Comment:
Removing options from FundFlowSelector component. They will be passed down as props from the parent component.